shell/common/rasterizer.h

```diff
@@ -20,78 +20,367 @@
 
 namespace flutter {
 
-/// Takes |LayerTree|s and draws its contents.
+//------------------------------------------------------------------------------
+/// The rasterizer is a component owned by the shell that resides on the GPU
+/// task runner. Each shell owns exactly one instance of a rasterizer. The
+/// rasterizer may only be created, used and collected on the GPU task runner.
+///
+/// The rasterizer owns the instance of the currently active on-screen render
+/// surface. On this surface, it renders the contents of layer trees submitted
+/// to it by the engine (which lives on the UI task runner).
```

Comment:
Maybe explicitly specify `flutter::Engine` so readers won't be confused with the engine as a whole (e.g., flutter/engine repo).

Reply:
Done.